docs/configure.rst

```diff
@@ -20,6 +20,14 @@ Environment variables
    :case: upper
 
 
+Alembic requires an additional item in the environment:
+
+.. code-block:: bash
+
+   # URI for user with ddl access
+   SQLALCHEMY_URL=mysql+pymysql://USER:PASSWORD@HOST:PORT/DBNAME
```

Comment:
I verified the variable is `SQLALCHEMY_URL` as opposed to `...URI`.

I also verified the structure is correct.

Having said that, the user doesn't need ddl access, so I'll fix the comment.